src/bot/checks.rs

```diff
@@ -6,10 +7,24 @@ use tracing::error;
 use crate::bot::Context;
 use crate::mysql_lib;
 
+static MAIN_GUILD_ID: Lazy<Option<u64>> = Lazy::new(|| match env::var("MAIN_GUILD_ID") {
+    Ok(val) => match val.parse() {
+        Ok(val) => Some(val),
+        Err(err) => {
+            error!(error = err.to_string(), "Failed to parse int");
+            None
+        }
+    },
+    Err(_) => {
+        error!("No Main Guild ID given");
+        None
+    }
+});
+
```

Comment:
We can probably leave out the `Option<..>` here, src/logging.rs already panics if MAIN_GUILD_ID isn't set correctly, and the bot most likely needs a main guild in order to work. I personally don't think it makes sense to make the bot capable of starting without having a main guild.

Reply:
Yeah agree. I think the best would be to have a single `env` module where every needed env is checked at start, and that panics if not present. I can do that once both PR are merged

Reply:
Might be a good idea to use something like the [config](https://crates.io/crates/config) crate or [dotenvy](https://crates.io/crates/dotenvy), then we'd have even have a way to store config that is permanent.

Reply:
Both of these are not really needed in our case, we don't actually want to change any of the values of the env variables at runtime.

For our limited usage of env variable, an own module is fine